custom/enikshay/two_b_release_1/management/commands/enikshay_2b_treatment_status_fix.py

```diff
@@ -0,0 +1,79 @@
+import datetime
+import logging
+
+from corehq.apps.es import CaseSearchES
+from django.core.management import BaseCommand
+from casexml.apps.case.mock import CaseStructure, CaseFactory
+
+logger = logging.getLogger('enikshay_2b_reason_for_test_fix')
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'domain',
+            help="The domain to migrate."
+        )
+        parser.add_argument(
+            "log_path",
+            help="Path to write the log to"
+        )
+        parser.add_argument(
+            '--commit',
+            action='store_true',
+            help="actually modifies the cases. Without this flag, it's a dry run."
+        )
+
+    def handle(self, domain, log_path, **options):
+        commit = options['commit']
+        factory = CaseFactory(domain)
+
+        logger.info("Starting {} migration on {} at {}".format(
+            "real" if commit else "fake", domain, datetime.datetime.utcnow()
+        ))
+
+        cases = (CaseSearchES()
+                 .domain(domain)
+                 .case_type("episode")
+                 .case_property_query("updated_by_migration", "enikshay_2b_case_properties", "must")
+                 .run().hits)
```

Comment:
Have you confirmed that this is a relatively small number of cases? We might need to paginate the query if not. One way to confirm the size is to do `query.count()` in a shell (instead of `query.run().hits`).